packages/server/test/Dishes.test.ts

```diff
@@ -8,14 +8,32 @@ import { initE2eFixture } from './E2eFixture';
 import { accessTokenAsCookie } from './shared';
 
 const PATH = '/api/partner/dishes';
+const PATH_ALL_DISHES = '/api/dishes';
 
 describe(`${PATH}`, () => {
   const fixture = initE2eFixture();
 
+  it('GET /', async () => {
```

Comment:
Fajnie byłoby to odseparować, bo teraz jak padnie ten test, to będzie, że nie działa `/api/partner/dishes GET /`, a pytamy w teście zupełnie inny path - `GET /api/dishes`.

Reply:
W #66 testy są w `PartnerDishes.tests.ts` - to chyba rozwiązuje problem? 🤔

Reply:
No jeśli w `PartnerDishes` będą testy `/api/partner/dishes`, a tu w `Dishes` testy `/api/dishes` będzie ok. Chodzi mi głównie o to, żeby były osobne bloki `describe` dla osobnych URL endpointu, bo potem przy failującym teście będą złe wiadomości błędu.